test/util.ts

```diff
@@ -1092,33 +1093,6 @@ describe('common/util', () => {
       };
     }
 
-    describe('callback mode', () => {
-      it('should pass the default options to retryRequest', (done) => {
```

Comment:
These tests are already in the `describe('callback mode` that exists below the `describe('stream mode`.